nemoguardrails/llm/prompts/openai-chatgpt.yml

```diff
@@ -222,7 +222,7 @@ prompts:
         content: "This is the current conversation between the user and the bot:"
       - "{{ history | colang | to_messages_v2}}"
 
-      - type: assistant
+      - type: system
```

Comment:
are any other prompt templates (for other models) affected as well? Or they were already updated in the past?

Reply:
Currently, we only support openai and llama3 models with the new interface. So, these are the only ones so far.

Reply:
OK got it.